README.md

```diff
@@ -239,16 +239,11 @@ to download dependencies first.
 #### Build steps <a name="build-steps"></a>
 
 1. Install the dependencies
-
-`npm install`
-
+    `npm install`
```

Comment:
Overall I don't like the idea of indentating those sections as it does not work well for bigger contents. Here it kind of looks fine since there isn't much content.

Reply:
Well, it's md syntax. Ie if you want to write a list, you should indent the lines that follow the bullet/number, otherwise the list will restart every time.